indico/modules/auth/templates/accounts.html

```diff
@@ -148,6 +148,22 @@
                 {% endif %}
             </div>
         </div>
+
+        <div class="i-box">
+            <div class="i-box-header">
+                <div class="i-box-title">
+                    {%- trans %}Sessions{% endtrans -%}
+                </div>
+            </div>
+            <div class="i-box-content">
+                <p>
+                    {% trans %}Invalidate all other sessions.{% endtrans %}
+                </p>
+                <button type="button" class="i-button highlight" data-href="{{ url_for('auth.sessions_invalidate') }}" data-method="post">
```

Comment:
Yes, I too felt this needed a little more (or clearer) explanation, I'd kind of taken my lead from the other screens in the user settings/preferences section where there is minimal contextual help. I'll add the title as suggested.